{"uuid":"343c5d40-2892-40cc-9bd5-c8ef1148fe38","historyId":"697be5c623328828f28b2546da814505","testCaseId":"[engine:junit-jupiter]/[class:com.Rently.Business.Integration.UsuarioServiceImplIntegrationTest]/[method:create_InvalidPhone_ShouldThrowException()]","testCaseName":"CREATE - Teléfono inválido debe lanzar excepción","fullName":"com.Rently.Business.Integration.UsuarioServiceImplIntegrationTest.create_InvalidPhone_ShouldThrowException","labels":[{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:com.Rently.Business.Integration.UsuarioServiceImplIntegrationTest]/[method:create_InvalidPhone_ShouldThrowException()]"},{"name":"host","value":"Laumcs"},{"name":"thread","value":"24384@Laumcs.main(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"com.Rently.Business.Integration.UsuarioServiceImplIntegrationTest"},{"name":"testClass","value":"com.Rently.Business.Integration.UsuarioServiceImplIntegrationTest"},{"name":"testMethod","value":"create_InvalidPhone_ShouldThrowException"},{"name":"suite","value":"UsuarioService - Integration Tests"}],"links":[],"name":"CREATE - Teléfono inválido debe lanzar excepción","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1759800786508,"stop":1759800786736}